slam-libraries/viam-cartographer/src/slam_service/config.cc

```diff
@@ -0,0 +1,90 @@
+// This is an Experimental variation of cartographer. It has not yet been
+// integrated into RDK.
+#include "config.h"
+
+#include <iostream>
+
+#include "glog/logging.h"
+
+// Functions that need to be implemented:
```

Comment:
Will these functions go in config? I would expect them to go in the grpc server.

Reply:
oh yeah good catch, doesn't belong in here.